InformationScripting/src/graph/PropertyMap.cpp

```diff
@@ -41,4 +47,19 @@ Property InformationScripting::PropertyMap::operator[](const QString& key) const
 	Q_ASSERT(false);
 }
 
+Property& InformationScripting::PropertyMap::operator[](const QString& key)
+{
+	for (auto& content : properties_)
+		if (content.first == key) return content.second;
+	qDebug() << "No object with name" << key;
```

Comment:
Do a typo in the key you type in [...] and you get a default property, somewhere down the road the program will probably crash, then I think it could be hard to track back to this.

I suggest to remove the Q_ASSERT() and return a default Property, but to leave the qDebug statement, with a TODO to remove it later